### DISCUSSION TITLE
请问 semi 是否支持nextjs 服务端渲染？

### DISCUSSION THREAD

**Discussion by 262146150** [DouyinFE/semi-design#1933] [General]
文档说支持SSR，但是在不加 use client 的情况下，报错，难道是我使用的姿势不对吗？

**Comment by pointhalo** [comment]
推荐先查一下  Server Side Render  与 React Server Component的概念，两者不是一码事。

**Reply by 262146150** [reply]
感谢， 瞬间明白了
